pkg/envoy/bootstrap/config.go

```diff
@@ -278,6 +278,11 @@ func (b *Builder) Build() (*xds_bootstrap.Bootstrap, error) {
 							},
 						},
 					},
+					UpstreamConnectionOptions: &xds_cluster.UpstreamConnectionOptions{
+						TcpKeepalive: &xds_core.TcpKeepalive{
+							KeepaliveTime: uint32(60),
```

Comment:
@steeling we are additionally able to define the keepalive `probes` and `interval` values, does this seem necessary? We haven't defined these for grpc because go's grpc pkg doesn't have these options. If not they will default as follows:

<img width="588" alt="image" src="https://user-images.githubusercontent.com/42152676/193928288-b82d2e76-f747-475a-b730-93294844abcb.png">